hips/draw/simple.py

```diff
@@ -178,6 +179,40 @@ def draw_tiles(self) -> np.ndarray:
 
         return image
 
+    def draw_markers_tiles(self, tile_index: int) -> None:
+        """Draw markers on the output image.
+
+        The following denotes their correspondence:
+        * red <=> North
+        * green <=> West
+        * blue <=> South
+        * yellow <=> East
+
+        Parameters
+        ----------
+        tile_index : int
+            HiPS tile index
+        """
+        try:
+            radec = self.tiles[tile_index].meta.skycoord_corners
+            colors = ['red', 'green', 'blue', 'yellow']
+            ax = plt.subplot(projection=self.geometry.wcs)
+            for i in range(len(radec.data.lon.deg)):
+                ax.scatter(radec.data.lon.deg[i], radec.data.lat.deg[i], s=80, transform=ax.get_transform('icrs'), color=colors[i])
+            ax.imshow(self.image, origin='lower')
+            plt.show()
+        except IndexError:
+            raise IndexError(f'Index: {tile_index} is out of bounds')
+
+    def draw_lines_tiles(self) -> None:
+        """Draw lines on the output image."""
+        for tile in self.tiles:
+            radec = tile.meta.skycoord_corners
+            ax = plt.subplot(projection=self.geometry.wcs)
+            ax.plot(radec.data.lon.deg, radec.data.lat.deg, 'red', lw=1, transform=ax.get_transform('icrs'))
+        ax.imshow(self.image, origin='lower')
+        plt.show()
```

Comment:
You should remove the `plt.show` call here and instead let users do that call if needed.
This will prevent the hanging of the tests at https://travis-ci.org/hipspy/hips/jobs/254765525#L1215 because a GUI window pops up, and it will also prevent a "double plotting" bug in Jupyter notebooks, where if people are using the common `%matplotlib inline`, the Figure will be plotted twice if you call `plt.show` here.